initialbuild.sh

```diff
@@ -0,0 +1,8 @@
+#bash 
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX:PATH="/workspace/Sample-Server/install" ../.github/ 
```

Comment:
Does gitpod known any enviroment variables? Those hardcoded directories ('/workspace/Sample-Server/') are always critical.

Reply:
This is true. Updated

Reply:
Hope this is ok, @ccvca 
https://github.com/umati/Sample-Server/pull/51/commits/2797bc27e8e2cfda837980eb7a308231f1116d0a